common/config/rush/build-cache.json

```diff
@@ -1,5 +1,12 @@
 {
     "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/build-cache.schema.json",
     "buildCacheEnabled": false,
-    "cacheProvider": "local-only"
-}
+    "cacheProvider": "amazon-s3",
+    "cacheEntryNamePattern": "[projectName:normalize]-[phaseName:normalize]-[hash]",
+    "amazonS3Configuration": {
+      "s3Bucket": "gdc-github-cache",
+      "s3Region": "us-east-1",
+      "s3Prefix": "cache/gooddata/gooddata-sdk/build-cache",
```

Comment:
not sure if this is already created somewhere, but if not and if possible, I would consider using `gooddata-ui-sdk` instead of `gooddata-sdk`

Reply:
Fixed